.github/workflows/docker.yml

```diff
@@ -39,84 +54,181 @@ on:
 
       - "tagging/**"
       - "tests/**"
-      - "Makefile"
       - "requirements-dev.txt"
   workflow_dispatch:
 
+# https://docs.github.com/en/actions/using-jobs/using-concurrency
 concurrency:
-  # only cancel in-progress jobs or runs for the current workflow - matches against branch & tags
```

Comment:
can we keep the comment? - it helps folks to understand what is being done here if they are less familiar with GH actions